modules/testsuite/cxf-tests/pom.xml

```diff
@@ -96,6 +96,12 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+        <groupId>org.wildfly.security</groupId>
+        <artifactId>wildfly-elytron-client</artifactId>
+        <version>1.10.0.CR5</version>
```

Comment:
Please parametrize the version via property.

Reply:
fixed